tracer/src/Datadog.Trace/Activity/Handlers/IgnoreActivityHandler.cs

```diff
@@ -0,0 +1,78 @@
+// <copyright file="IgnoreActivityHandler.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#nullable enable
+
+using Datadog.Trace.Activity.DuckTypes;
+
+namespace Datadog.Trace.Activity.Handlers
+{
+    /// <summary>
+    /// Ignore Activity Handler catches existing integrations that also emits activities.
+    /// </summary>
+    internal class IgnoreActivityHandler : IActivityHandler
+    {
+        private static readonly string[] SourcesNames =
+        {
+            "Couchbase.DotnetSdk.RequestTracer",
```

Comment:
Elasticsearch 8 has diagnostic source I think: https://www.elastic.co/guide/en/elasticsearch/client/net-api/current/diagnostic-source.html

I wonder if we have a way to detect if later versions of packages add support for diagnostic source, so we know to turn them off here? 🤔 We would detect it with snapshots, but not everything has snapshots...

Reply:
I'm not sure about automatic detection, for this case in specific I think we can add the keys from: https://github.com/elastic/elasticsearch-net/blob/7.17/src/Elasticsearch.Net/Diagnostics/DiagnosticSources.cs#L70-L95

Reply:
> I'm not sure about automatic detection

I guess the best answer is for us to transition to snapshots wherever possible, as they _will_ pick up things like this

Reply:
Npgsql and MySQL also added Activity support in newer versions, afaik.